daemon/container.h

```diff
@@ -123,10 +123,10 @@ container_t *
 container_new(const uuid_t *uuid, const char *name, compartment_type_t type, bool ns_usr,
 	      bool ns_net, const void *os, const char *config_filename, const char *images_folder,
 	      unsigned int ram_limit, const char *cpus_allowed, uint32_t color,
-	      bool allow_autostart, const char *dns_server, list_t *net_ifaces,
-	      char **allowed_devices, char **assigned_devices, list_t *vnet_cfg_list,
-	      list_t *usbdev_list, const char *init, char **init_argv, char **init_env,
-	      size_t init_env_len, list_t *fifo_list, container_token_type_t ttype,
+	      bool allow_autostart, unsigned int global_resources, const char *dns_server,
```

Comment:
refactor bool ns_user, ns_net and allow_autostart as container_flags first,
then introduce flag for clock_realtime.
uint64_t container_flags should include COMPARTMENT_NS_USER, COMPARTMENT_NS_NET, COMPARTMENT_AUTOSTART, COMPARTMENT_SYSTEM_CLOCK_REALTIME